src/core/include/openvino/core/parallel.hpp

```diff
@@ -22,6 +22,18 @@
 #define OV_THREAD_SEQ      2
 #define OV_THREAD_TBB_AUTO 3
 
+#if (OV_THREAD == OV_THREAD_TBB)
+#    define OV_THREAD_NAME "TBB"
+#elif (OV_THREAD == OV_THREAD_OMP)
+#    define OV_THREAD_NAME "OMP"
+#elif (OV_THREAD == OV_THREAD_SEQ)
+#    define OV_THREAD_NAME "SEQ"
+#elif (OV_THREAD == OV_THREAD_TBB_AUTO)
+#    define OV_THREAD_NAME "TBB_AUTO"
+#else
+#    define OV_THREAD_NAME "UNKNOWN"
+#endif
```

Comment:
Is it requried for external user?
Looks like more as some info/debug message, can be moved as util `constexpr function/inline variable` for internal use only?

Reply:
Yes, this I wanted to discuss.
Currently there is no way (at least I have not found it) to get threading runtime as a string, enum value or something else.
I though it will be useful to have it in the core file, but I can move it to cpu plugin specific file if necessary.

Reply:
@praasz What do you think?

Reply:
I guess we should throw in 'default' since it should not be possible

Reply:
Updated and moved to cpu plugin internal files for now